client/gutenberg/editor/media-utils.js

```diff
@@ -39,6 +39,15 @@ export const mediaCalypsoToGutenberg = ( media ) => {
 		type: get( media, 'mime_type', '' ).split( '/' )[ 0 ],
 		width: get( media, 'width' ),
 	};
+
+	// VideoPress data
+	if ( media.videopress_guid ) {
+		mediaData.allow_download = media.allow_download;
+		mediaData.rating = media.rating;
+		mediaData.videopress_guid = media.videopress_guid;
+	}
```

Comment:
Nit: if you'd like to avoid mutating objects (which we're effectively doing here), you can do something like:

```
const { allow_download, rating, videpress_guid } = media;
const videopressData = videopress_guid ? { allow_download, rating, videopress_guid } : {};

return {
	id: get( media, 'ID' ),
	// ....    
	...videopressData,
```

That's just optional since I usually try to avoid mutating whenever possible.

Reply:
Thanks for the explanation. Marin.